RATIONALE.md

```diff
@@ -432,6 +432,22 @@ The main difference between Jsonnet and Nickel are types. Jsonnet does not
 feature static types, contracts or metadata, and thus can't type library code
 and has no principled approach to data validation.
 
+### KCL vs Nickel
+
```

Comment:
Would that be possible to add a small sentence first to say that they have many things common as well, and that KCL seems to be a capable configuration language? (types, validation, modularity, merging, functions)